src/reactComponents/smart-avatar.js

```diff
@@ -12,22 +12,34 @@ export default class SmartWebexTeamsAvatar extends Component {
     super(props);
 
     this.personID = props.personID;
+    this._isMounted = false;
     this.adapter = props.adapter;
     this.state = {
       person: undefined
     };
   }
 
   componentDidMount() {
-    this.adapter.getPerson(this.personID).then((person) => {
-      this.updatePerson(person);
-    })
+    this._isMounted = true;
+    this.adapter.getPerson(this.personID)
```

Comment:
We should probably store this to a local variable so we can unsubscribe:

```js
this.getPersonSubscription = this.adapter.getPerson
```

then in `componentWillUnmount()`
```js
this.getPersonSubscription.unsubscribe();`
```